eng/Version.Details.xml

```diff
@@ -90,6 +90,10 @@
       <Uri>https://github.com/microsoft/vstest</Uri>
       <Sha>2418d9ec492e93245e1ce8699d6ae9cd7b86649b</Sha>
     </Dependency>
+    <Dependency Name="System.CommandLine" Version="2.0.0-beta1.20253.1" Pinned="true">
```

Comment:
This is also listed at: https://github.com/dotnet/runtime/blob/7e1ad2795b4c8f6b0085ddbbb477c807e587cca4/eng/Versions.props#L124 can we consolidate dependency versions in one file?